lib/service/servicecfg/plugins.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicecfg
+
+type PluginsConfig struct {
+	Enabled bool
+	// OpsgenieAPIKeyFile is a file containing an Opsgenie API key.
+	OpsgenieAPIKeyFile string
```

Comment:
I think that in the file config we can take path, but when converting it to the actual service config, we can read the file in and have the actual value here.